yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -201,41 +201,47 @@ func (adb *AssessmentDB) BulkInsert(table string, records [][]string) error {
 }
 
 const (
-	InsertTableStats = `INSERT INTO %s (schema_name, object_name, row_count, column_count, reads, writes, is_index, object_type, parent_table_name, size_in_bytes)
+	InsertTableStats = `INSERT INTO %s (schema_name, object_name, row_count, column_count, reads, writes, reads_per_second, writes_per_second, is_index, object_type, parent_table_name, size_in_bytes)
 	SELECT 
 		trc.schema_name,
 		trc.table_name AS object_name,
 		trc.row_count,
 		tcc.column_count,
-		tii.seq_reads AS reads,
-		tii.row_writes AS writes,
+		COALESCE(tii.seq_reads, 0) AS reads,
+		COALESCE(tii.row_writes, 0) AS writes,
+		0 AS reads_per_second,
+		0 as writes_per_second,
 		0 AS is_index,
 		otm.object_type,
 		NULL AS parent_table_name,
-		tis.size_in_bytes
+		COALESCE(tis.size_in_bytes, 0)
 	FROM %s trc
 	LEFT JOIN %s tii ON trc.schema_name = tii.schema_name AND trc.table_name = tii.object_name and tii.measurement_type='initial'
 	LEFT JOIN %s tis ON trc.schema_name = tis.schema_name AND trc.table_name = tis.object_name
 	LEFT JOIN %s tcc ON trc.schema_name = tcc.schema_name AND trc.table_name = tcc.object_name
-	LEFT JOIN %s otm ON trc.schema_name = otm.schema_name AND trc.table_name = otm.object_name
+	JOIN %s otm ON trc.schema_name = otm.schema_name AND trc.table_name = otm.object_name
```

Comment:
otm table have more information of object name -> type mapping, like it also contained sequences, some internal indexes in case oracle, which are not required in the final table etc... etc..

Reply:
but if you do a LEFT JOIN, it should only consider rows from left table (trc) which have matching rows in otm, right? so, that should be fine right?

Reply:
reverted the change

Reply:
Please update the description of the PR as well.